src/main/java/com/o19s/es/ltr/ranker/parser/XGBoostJsonParser.java

```diff
@@ -59,20 +59,32 @@ public NaiveAdditiveDecisionTree parse(FeatureSet set, String model) {
         Arrays.fill(weights, 1F);
         return new NaiveAdditiveDecisionTree(trees, weights, set.size(),
                 modelDefinition.normalizer,
-                modelDefinition.useFloatMaxForMissing ? Float.MAX_VALUE : null);
+                modelDefinition.missingValue);
     }
 
     private static class XGBoostDefinition {
         private static final ObjectParser<XGBoostDefinition, FeatureSet> PARSER;
         static {
             PARSER = new ObjectParser<>("xgboost_definition", XGBoostDefinition::new);
             PARSER.declareString(XGBoostDefinition::setNormalizer, new ParseField("objective"));
+
+            // Parameters to represent missing features in feature vector
+            //
+            // There are convenience options like `use_float_max_for_missing`
+            // which can be used instead of specifying the actual floating
+            // point value.
+            //
+            // Only one of the following parameter should be specified
+            // otherwise the behavior is undefined (depends on the order of
+            // names in JSON).
+            PARSER.declareFloat(XGBoostDefinition::setMissingValue, new ParseField("missing_value"));
```

Comment:
instead of adding a new param it might be more coherent to be able to parse a float and string
this way one could pass:
`"missing_value": "max"`
or
`"missing_value": 0.0`

everything would throw an exception.

See https://github.com/elastic/elasticsearch/blob/237650e9c054149fd08213b38a81a3666c1868e5/server/src/main/java/org/elasticsearch/search/suggest/completion/FuzzyOptions.java#L66 to declare a field that accepts any kind of value
and https://github.com/elastic/elasticsearch/blob/f92ebb2ff909d0083ae988e04ecd398d979e9210/server/src/main/java/org/elasticsearch/common/unit/Fuzziness.java#L160 for how to parse the value